lib/proxy/sockjs.js

```diff
@@ -29,7 +29,9 @@ function createServer(router, schedulerRegistry, reconnect) {
   var sockjsServer = sockjs.createServer({
     // TODO: make URL configurable
     sockjs_url: '//d1fxtkz8shb9d2.cloudfront.net/sockjs-0.3.min.js',
-    prefix: '.*/__sockjs__/[no]=\\w+',
+    prefix: reconnect ?
+      '.*/__sockjs__/[no]=\\w+' :
+      '.*/__sockjs__',
```

Comment:
If we can use a single prefix pattern for both cases, then we can make reconnection a per-app config property.